.github/workflows/Pitests.yml

```diff
@@ -0,0 +1,635 @@
+name: "Pitest"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  Pitest-main:
+    runs-on: macos-latest
```

Comment:
why macos ? is here, please use ubuntu, it is our kind of default OS.
ALL other OSs are mostly checking "mvn verify" only.

Reply:
Done.